tests/e2e/set-ffi-env-e2e

```diff
@@ -184,10 +184,20 @@ install_qm_rpms() {
   dnf install -y 'dnf-command(config-manager)'
   dnf config-manager --set-enabled crb
 
+  local release_id
+  release_id=$(grep ^ID= /etc/os-release | cut -d = -f 2)
+
   if [[ -n "${USE_QM_COPR}" && "${USE_QM_COPR}" != "release" ]]; then
       dnf copr enable -y @centos-automotive-sig/bluechi-snapshot centos-stream-9
       dnf copr enable -y "${USE_QM_COPR}" centos-stream-9
   fi
+
+  if [[ -n "${USE_QM_COPR}" && "${USE_QM_COPR}" == "release" && "${release_id}" =~ "centos" ]]; then
+      if [ -z "$(find /etc/yum.repos.d/autosd.repo -mindepth 1 -maxdepth 1)" ]; then
+          install_autosd_repo
```

Comment:
This one could be done as follow

```
if [ ! -f /etc/yum.repos.d/autosd.rep ]; then 
   install_autosd_repo
fi

``` 
 Please consider an info print
 info_message "Installing autosd_repo"